client/my-sites/stats/stats-module-devices/stats-module-devices.tsx

```diff
@@ -32,18 +35,54 @@ interface StatsModuleDevicesProps {
 	className?: string;
 	period?: string;
 	postId?: number;
-	query: object;
+	query: QueryStatsDevicesParams;
 	summary?: boolean;
 	isLoading?: boolean;
 }
 
+const prepareChartData = ( data: Array< StatsDevicesData > | undefined ) => {
+	if ( ! data ) {
+		return [];
+	}
+
+	return data.map( ( device: StatsDevicesData ) => {
+		let icon;
+
+		switch ( device.key ) {
+			case 'desktop':
+				icon =
```

Comment:
I think we had a file capturing all SVG icons, but I also remember we had some SVG issues with Odyssey. Do you think we could extract SVG to another file?

Reply:
There is a tricky issue with [updating the SVG as HTML to the tooltip](https://github.com/Automattic/wp-calypso/blob/e220fc39053d2c198a9b0b9d09ffbfc497bba243/client/components/pie-chart/index.js#L70-L71) by directly importing an existing file or component. I will try to find a solution to address the sharability.

Reply:
Maybe let's park this and address as a polish phase once we have everything else sorted.